libqtile/widget/systray.py

```diff
@@ -190,13 +198,30 @@ def draw(self):
                 0,
                 None
             )
+            if icon.hidden:
+                icon.unhide()
+                data = [
+                    self.qtile.conn.atoms["_XEMBED_EMBEDDED_NOTIFY"],
+                    xcffib.xproto.Time.CurrentTime,
+                    0,
+                    self.bar.window.window.wid,
+                    icon.protocol_version
+                ]
+                u = xcffib.xproto.ClientMessageData.synthetic(data, "I" * 5)
+                xcffib.xproto.ClientMessageEvent.synthetic(
```

Comment:
This event is never sent, you have to call `self.window.window.send_event()` on the event that `.synthetic()` return.